components/message/demo/close.ts

```diff
@@ -0,0 +1,27 @@
+import { Component, OnInit } from '@angular/core';
+import { NzMessageService } from 'ng-zorro-antd';
+
+@Component({
+  selector: 'nz-demo-message-close',
+  template: `
+    <button nz-button [nzType]="'default'" (click)="createBasicMessage()">Customized close callback</button>
+  `,
+  styles: []
+})
+export class NzDemoMessageCloseComponent implements OnInit {
+
+  ngOnInit(): void {
+    this.message.nzAfterClose.subscribe(message => {
```

Comment:
If a user has multiple message opening at the same time, how could s/he know which message is closed?